frontend/app/views/event/list.scala.html

```diff
@@ -8,6 +8,10 @@
 @import com.gu.membership.salesforce.Tier
 @import model.Benefits
 
+@emptyListing = {
+    <div class="listing__empty">@noResultsMessage</div>
```

Comment:
Although this is repeated, I actually prefer the repetition; my personal preference is to avoid anything that could accidentally cause a BEM fragment (child used without the parent). What do you think?